common/http.go

```diff
@@ -0,0 +1,131 @@
+package common
+
+import (
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"net/http"
+
+	"gopkg.in/go-playground/validator.v9"
+
+	"github.com/pkg/errors"
+)
+
+type ErrorResponse struct {
+	Success bool `json:"success,omitempty"`
```

Comment:
So I am keeping the omitempty so that in cases where we do
`success` and leave the rest out, only `success` is sent on the wire and the opposite, when `err` and `code` are defined, we only send `err` and `code` on the wire.